app/settings/production.py

```diff
@@ -34,7 +34,7 @@
     ANYMAIL = {
         "SEND_DEFAULTS": {"envelope_sender": DEFAULT_FROM_EMAIL},
         "MAILJET_API_KEY": os.getenv("MAILJET_API_KEY"),
-        "MAILJET_SECRET_KEY": os.getenv("MAILJET_SECRET_KEY"),
+        "MAILJET_SECRET_KEY": os.getenv("MAILJET_SECRET π_KEY"),
```

Comment:
Typo?

Reply:
Ah yes, have been waiting for the domain to transfer before configuring mailgun